NEMO/templates/tool_control/tool_status.html

```diff
@@ -245,6 +245,23 @@ <h4>What would you like to do?</h4>
 				{% endif %}
 			{% endif %}
 		</form>
+		<div style="background-color: #f5f5f5 ; ">
+
+			<div class="row" style="margin:0">
+				{% if tool.description %}
+				<div class="col-md-8">
+					<p> {{tool.description| safe }} </p>
+				</div>
+				{% endif %}
+				{% if tool.image %}
+				<div class="col-md-4">
+					<center><p><img src="{{ tool.image.url }}" class="img-fluid img-thumbnail float-right" style="max-width:300px ;max-height:300px" ></p></center>
```

Comment:
`<center>` is a deprecated tag. I added `text-align: center` instead in my suggestion below